src/cli/src/lib.rs

```diff
@@ -173,6 +179,64 @@ impl RootCmd {
     }
 }
 
+// For some reason this doc string is the one that gets used for the top-level help menu.
+// Please don't change it unless you want to update the top-level help menu "about".
+/// Fluvio command-line interface
+#[derive(StructOpt, Debug)]
+pub enum FluvioCmd {
```

Comment:
Just FYI this is not all new code, I just moved it here from `src/cli/consumer/mod.rs`